ai21_tokenizer/tokenizer_factory.py

```diff
@@ -2,12 +2,12 @@
 from pathlib import Path
 
 from ai21_tokenizer.base_tokenizer import BaseTokenizer
-from ai21_tokenizer.jamba_instruct_tokenizer import JambaInstructTokenizer
-from ai21_tokenizer.jurassic_tokenizer import JurassicTokenizer
+from ai21_tokenizer.jamba_instruct_tokenizer import JambaInstructTokenizer, AsyncJambaInstructTokenizer
+from ai21_tokenizer.jurassic_tokenizer import JurassicTokenizer, AsyncJurassicTokenizer
 
 _LOCAL_RESOURCES_PATH = Path(__file__).parent / "resources"
 _ENV_CACHE_DIR_KEY = "AI21_TOKENIZER_CACHE_DIR"
-JAMABA_TOKENIZER_HF_PATH = "ai21labs/Jamba-v0.1"
+JAMBA_TOKENIZER_HF_PATH = "ai21labs/Jamba-v0.1"
```

Comment:
In the `PreTrainedTokenizers.JAMBA_INSTRUCT_TOKENIZER ` the string is wrong isn't? It should be `jamba-tokenizer`?

Reply:
(not your fault but should be changed)